ex40.py

```diff
@@ -0,0 +1,22 @@
+import module
+
+print "Module example"
+module.printName("Priti")
+print module.x
+
+class MyTest:
+	def __init__(self, x, y):
+		self.x = x
+		self.y = y
+		
+	def printValues(self):
+		x = 9
+		print "X = %r" % self.x
+		print "Y = %r" % self.y
+		print "local variable = %r" % x
```

Comment:
Use proper (single tab or 4 space in sublime text) indentations while defining functions and classes.